### ISSUE TITLE
Page landmarks should be tagged and described for assestive technology

### ISSUE THREAD

**Issue by hauf-toni** [digitalfabrik/integreat-app#3637]
### Describe Your Preferred Solution

Landmarks like main content and footer should use their corresponding HTML5 elements (<main>, <footer>) or include an "aria-label" and "role" attribute (e.g., role="main" or role="contentinfo"). Landmarks like search and navigation are covered elsewhere.

### Describe Possible Alternatives

/

### Related Issues

/

### Additional Information

Improves overall accessibilty score.

**Comment by hauf-toni** [comment]
@steffenkleinle fine with me
